qutebrowser/config/configdata.yml

```diff
@@ -432,7 +433,7 @@ content.javascript.can_access_clipboard:
   default: false
   type: Bool
   desc: >-
-    Whether JavaScript can read from or write to the clipboard.
+    Enable JavaScript to read from or write to the clipboard.
```

Comment:
The "Enable javascript to <x>" phrase feels a little weird to me, not sure why.

I think that "Allow javascript to <x>" **might** be better, but it would sacrifice the consistency of naming for boolean values. I'm open to both, just wanted to bring it up.